server/service/ResultService.ts

```diff
@@ -188,7 +188,24 @@ const service = {
   getClub: async ({ year, limit }: Partial<OptionsYearLimitRegion>) => {
     const seasonDetail = await retrieveSeasonDetails(year);
 
-    const constantsForResult = { NUMBER_OF_SCORED_FLIGHTS };
+    // Determines how many flights per club member will be scored.
+    // Use global default if no value was found for that particular season.
+    const NUMBER_OF_SCORED_FLIGHTS_CLUB =
```

Comment:
Hm bei mir sind es die 30 besten und max drei pro pilot, egal was ich in den seasondetails festlege.

Reply:
Hast du ein DB seed gemacht?

Reply:
Ja, habe auch den Text zu testen verändert und die Änderung ist sichtbar.

Reply:
Also bei mir funktioniert es in allen erdenklich Optionen. Evtl. ein Tippfehler bei der Number?

Reply:
@JEK58 konntest du das nochmal prüfen?